book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd

```diff
@@ -128,7 +130,13 @@ This will not go into much detail about about how to use Docker -- for more deta
         Done!
         ###################################################################### 
         ```
-
+    e. Download the [`pecan/docker/env.example`](https://raw.githubusercontent.com/PecanProject/pecan/develop/docker-compose.yml) & save it as `.env` file.
+       Now, open the `.env` file & uncomment the lines mentioned below:
```

Comment:
I liked the idea of showing this visually, but I think documentation should explain why we need to uncomment traefik configuration flags (I personally don't uncomment them)

Reply:
Ok...will speak with @robkooper and write about the explanation for uncommenting these lines

Reply:
I agree, we can do that in a later section about configuring traefik. This should be just about getting pecan up and running.

Reply:
So...I suppose we can leave this part as is? And include details about traefik later in the documentation @istfer ?

Reply:
by "as is" do you mean leaving the PR as is, with the screenshot that `TRAEFIK` flags uncommented? I thought @robkooper also agreed that this bit shouldn't involve anything related to traefik configuration

Reply:
I thought @robkooper mentioned that we show the user which lines to uncomment (using this image), and in a later part of the documentation , explain about traefik configurations.

Coz if we don't show the user what to uncomment at this stage of the installation, I'm afraid (s)he will have trouble in this stage of installation.

Reply:
I agree that it is nice to show which lines to uncomment, but I never uncomment TRAEFIK tags for my own set up (which might be something I'm doing wrong). That's why I was curious if this is something everyone must do, if yes it needs explanation, if not the screenshot is a bit misleading at this stage.

Overall, I'm with it staying as well.

Reply:
I think it would be best to only highlight the minimal work that needs to be done, in this case 2 variables
```
COMPOSE_PROJECT_NAME=pecan
PECAN_VERSION=develop
```
I think we should clarify that only set `PECAN_VERSION` if you want to run the develop branch, meaning it can have bugs.

Reply:
I have made required modifications...pls have a look @istfer @robkooper